index.html

```diff
@@ -0,0 +1,218 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Artificial Intelligence Conferernce</title>
+
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+
+    <link
+      rel="stylesheet"
+      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+      crossorigin="anonymous"
+    />
+
+    <link rel="stylesheet" href="./css/styles.css" />
+    <script defer src="./js/index.js"></script>
+    <script defer src="./js/mobile_menu.js"></script>
+  </head>
+  <body>
+    <header>
+      <ul class="top">
+        <li class="top-item">
+          <a href="#"><i class="fab fa-facebook-f"></i></a>
+        </li>
+        <li class="top-item">
+          <a href="#"><i class="fab fa-twitter"></i></a>
+        </li>
+        <li class="top-item">
+          <a href="#">English</a>
+        </li>
+        <li class="top-item">
+          <a href="#">Logout</a>
+        </li>
+        <li class="top-item">
+          <a href="#">MyPage</a>
+        </li>
+      </ul>
+    </header>
+    
+    <span class="hamburger"><i class="fas fa-bars"></i></span>
+    <nav class="navbar">
```

Comment:
The navbar should have the same design for both `About` and `Home` pages on the desktop.
Your:
![nav-y](https://user-images.githubusercontent.com/35504149/137373634-842362a7-dc18-4df4-bc2f-b9de2c354f0e.png)
Design:
![nav-o](https://user-images.githubusercontent.com/35504149/137373675-0c37f839-af18-457c-af87-84ebc9bc0f18.png)